packages/fluentui/react-northstar/src/themes/teams/components/List/listItemStyles.ts

```diff
@@ -62,7 +62,7 @@ const listItemStyles: ComponentSlotStylesPrepared<ListItemStylesProps, ListItemV
         ...(p.selected && selectedStyle(v)),
       }),
       ...(p.important && {
-        fontWeight: 'bold',
+        fontWeight: siteVariables.fontWeightBold,
```

Comment:
Let's avoid usage of `siteVariables` inside `*style`, let's map to it `importantFontWeight` in `listVariables` instead.

Reply:
fixed